bundle/src/main/java/com/adobe/acs/commons/properties/PropertyConfigService.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2020 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.properties;
+
+public interface PropertyConfigService {
+
+    /**
+     * Checks whether the passed property name should be excluded or not. This check is based on the OSGi
+     * configuration for the service.
+     *
+     * @param propertyName current property name
+     * @return whether to exclude or not
+     */
+    boolean isNotExcluded(final String propertyName);
```

Comment:
Fair enough - I typically follow this pattern with exclusion because:
A) IntelliJ has a warning saying 'the result of this method is always negated, you should consider changing the return value'
B) it's easier (when reading code) to see `isNot` instead of `!is` 

I'll change it to `isAllowed()` though - best of both worlds 😄